openedx/core/djangoapps/catalog/utils.py

```diff
@@ -86,7 +87,7 @@ def check_catalog_integration_and_get_user(error_message_field):
 
 
 # pylint: disable=redefined-outer-name
-def get_programs(site=None, uuid=None, uuids=None, course=None):
+def get_programs(site=None, uuid=None, uuids=None, course=None, organization=None):
```

Comment:
hmm, do we really need to add an organization kwarg to this if you're also going to provide the `get_programs_for_organization()` function below?

Reply:
I figured I'd just make this function work with organization as well since it was trivial.  The helper function only returns a list of program uuids.